### DISCUSSION TITLE
Manga Tagger "Failed to execute script".

### DISCUSSION THREAD

**Discussion by cecoates** [Inpacchi/Manga-Tagger#41] [Q&A]
I'm attempting to follow the steps outlined in https://github.com/Inpacchi/Manga-Tagger/wiki/Alpha-Installation-(Windows) for the first time. Apologies if I missed something obvious.

![Screenshot 2022-02-11 001517](https://user-images.githubusercontent.com/10121835/153545566-71606470-8fa9-40b8-85f4-c1dd18cb7a86.png)

[MangaTagger.log](https://github.com/Inpacchi/Manga-Tagger/files/8046300/MangaTagger.log)

The nssm Manga Tagger service also appears to be paused, with startup type automatic. Is that right?

If I try to stop and start it again it says "manga_tagger: Unexpected status SERVICE_PAUSED in response to START control.  "

**Comment by RedPhanthom** [comment]
I am also on the same boat as you, has there been any solution yet?
